fe/fe-core/src/main/java/com/starrocks/catalog/OlapTable.java

```diff
@@ -200,6 +204,8 @@ public enum OlapTableState {
     // that is, all transactions which id is smaller than binlogTxnId have been finished/aborted,
     // then binlog is available
     protected long binlogTxnId = -1;
+    @SerializedName(value = "abortDelete")
+    protected boolean abortDelete = false;
```

Comment:
It seems this property can be infered by schema, and schema maybe changing, so do we need to persist it? Also, consider what happens if user alter table(add/drop auto increment column)

Reply:
now abortDelete is defined in Class Table and persist it. For simplicity, we can only drop auto increment columns but can not add a new one.